.github/workflows/build.yml

```
@@ -0,0 +1,38 @@
+name: instrumentation_tests
+
+on: [push, pull_request]
+
+env:
+  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx4g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
```

```suggestion
  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx4g -Dorg.gradle.daemon=false -Dkotlin.incremental=false -Dorg.gradle.parallel=true"
```

Making this a parallel build should speed things up at zero cost to us: https://docs.gradle.org/current/userguide/performance.html#parallel_execution